vip_hci/pca/pca_fullfr.py

```diff
@@ -774,19 +774,19 @@ def _adi_rdi_pca(cube, cube_ref, angle_list, ncomp, scaling, mask_center_px,
     """ Handles the ADI+RDI post-processing.
     """
     n, y, x = cube.shape
-
+    n_ref, y_ref, x_ref = cube_ref.shape
     angle_list = check_pa_vector(angle_list)
     if not isinstance(ncomp, int):
         raise TypeError("`ncomp` must be an int in the ADI+RDI case")
-    if ncomp > n:
-        ncomp = min(ncomp, n)
+    if ncomp > n_ref:
+        ncomp = min(ncomp, n_ref)
         msg = 'Number of PCs too high (max PCs={}), using {} PCs instead.'
         print(msg.format(n, ncomp))
```

Comment:
Thanks for spotting this @jmilou. Could you also replace `n` by `n_ref` at line 784? Thanks!

Reply:
resolved in following PR (#453)